.asf.yaml

```diff
@@ -32,7 +32,6 @@ github:
   collaborators:
     - sullis
     - yashmayya
-    - abhioncbr
```

Comment:
I'd suggest keeping the collaborators list full (up to the maximum 10 allowed by the ASF). As per the output of `git shortlog --numbered --summary --since="1 year ago"` on `master`, the next highest contributor who can be added here is @deepthi912.

Reply:
Sure, it makes sense.